src/terra-dev-site/app/Application.1.app.mdx

```diff
@@ -0,0 +1,106 @@
+- How will this application have a user experience that is consistent with that of other applications? How will that consistency be maintained over time?
+
+The terra-application framework seeks to answer these questions and simplify the process of maintaining web applications.
+
+## Overview
+
+Terra Application contains many components designed to help solve common web application problems.
+
+### Application Base
+
+The ApplicationBase component is the entrypoint into the Terra application framework. Applications must render ApplicationBase around the their content to provide the content with ApplicationBase's features.
+
+[Application Base API](/application/terra-application/components/application-base)
+
+### Application Navigation
+
+Consistent navigation throughout Cerner applications helps provide a positive user experience by teaching users what to expect when interacting with applications. The ApplicationNavigation component provides a styled layout and controls used for navigating within an application.
+
+[Application Navigation API](/application/terra-application/components/application-navigation)
+
+### JS Error Handling
+
+Typically when a javascript error occurs in a web application, the entire page will be broken. Using [react error boundaries](https://reactjs.org/docs/error-boundaries.html), Terra provides a way to isolate the error to prevent the entire application from being taken down.
+
+[Application Error Boundary API](/application/terra-application/components/application-error-boundary)
+
+### Overlays
+
+Presenting overlays over content being loaded or saved to a server is a common application pattern. Terra provides an application overlay to overlay loading content, as well as an application overlay provider to redefine what portion of the page should be overlaid.
+
+[Application Overlay API](/application/terra-application/components/application-loading-overlay)
+[Application Overlay Provider API](/application/terra-application/components/application-loading-overlay-provider)
+
+### Modal disclosure
+
+Presenting content within a modal is a common, yet tricky pattern. To help reduce this complexity Terra provides API's, through the disclosure manager, to present modal content.
+
+[Disclosure Manager API](/application/terra-application/contexts/disclosure-manager-context)
+
+### Unsaved Changes
+
+Unsaved form data is another common pattern. Terra provides the Navigation prompt to prevent navigating away from unsaved changes.
+
+[Navigation Prompt API](/application/terra-application/components/navigation-prompt)
+
+### Slide panel disclosure
+
+Presenting content to a slide panel is less common, yet still tricky. Similar to modal manager we provide the disclosure manager api to disclose content to slide panels, as well as a slide panel manager to define the slide panel in the application.
+
+[Slide Panel Manager](/application/terra-application/components/slide-panel-manager)
```

Comment:
Should "API" be added to the name to be consistent with the rest?

These are displayed side by side. Can they be top bottom for readibility?
![Screen Shot 2020-04-27 at 10 35 29 AM](https://user-images.githubusercontent.com/22618655/80391299-2618d680-8873-11ea-924c-dcb7ec885873.png)

Reply:
I left off api on slide panel manager, because it's more of a component than an api? since they have to provide the component itself instead of like modal which is provided for them... it's an odd duck